### ISSUE TITLE
Null loss value after a while of smooth optimization 

### ISSUE THREAD

**Issue by erogol** [Lasagne/Lasagne#241]
Here is the output that I got ;

   3       0.52286       0.47760      1.09477      0.62482  431.44s
      4       0.43941       0.41642      1.05520      0.62473  427.18s
      5       0.38904       0.37730      1.03111      0.62469  426.38s
      6       0.35606       0.35055      1.01573      0.62461  423.12s
      7       0.33325       0.33144      1.00546      0.62462  419.26s
      8       0.31728       0.31721      1.00023      0.62465  420.03s
      9       0.30535       0.30584      0.99841      0.62472  418.48s
     10       0.29652       0.29630      1.00077      0.62489  419.86s
     11       0.29002       0.28839      1.00566      0.62501  419.37s
     12           nan           nan          nan      0.62500  419.85s
     13           nan           nan          nan      0.62500  419.30s
     14           nan           nan          nan      0.62500  421.47s
     15           nan           nan          nan      0.62500  420.63s

I am using nesterov momentum. Why do you think I observe such problem?

**Comment by benanne** [comment]
We will need a lot more information than just some output of your script. Without any additional info, my guess is that the learning rate is too high. If you are using cross entropy loss, make sure to truncate the predictions at [epsilon, 1-epsilon] to prevent it from becoming infinite.

Support requests should be posted on the mailing list by the way: https://groups.google.com/forum/#!forum/lasagne-users
Please reserve the GitHub issue queue for bug reports, feature requests etc.
